### PR TITLE
Create .env file configuration and profiles for dev/prod/test branches#16

### PR DESCRIPTION
**Pull request by PanfiDen**
Create .env file configuration and profiles for master/dev/test branches#16 
Also add Environment variable: "DB_LOGIN=sa;DB_PASSWORD=;DB_URL=jdbc:h2:mem:SkillScopeDB;SPRING_PROFILES_ACTIVE=dev"
For change the profile, write instead of SPRING_PROFILES_ACTIVE=dev: SPRING_PROFILES_ACTIVE=prod or SPRING_PROFILES_ACTIVE=test

### CODE REVIEW

**Review by IlliaKyselov1902 — APPROVED** [review]

**Review by SEM24 — APPROVED** [review]

File: src/main/resources/application-prod.properties
Comment (SEM24) [inline src/main/resources/application-prod.properties:1]: Probably better to change like this:
``` 
spring.datasource.url=${DB_URL}
spring.datasource.username=${DB_LOGIN}
spring.datasource.password=${DB_PASSWORD}

File: src/main/resources/application-test.properties
Comment (SEM24) [inline src/main/resources/application-test.properties:1]: Probably better to change like this or **_similar_**:
```
spring.datasource.username=sa
spring.datasource.password=
spring.datasource.url=jdbc:h2:mem:/testDB;DB_CLOSE_DELAY=-1

##PostgreSqlDBSetup##
#spring.datasource.url=jdbc:postgresql://localhost:5432/SkillScope
#spring.datasource.username=postgres
#spring.datasource.password=root

spring.h2.console.enabled=true
#spring.sql.init.mode=always
#
###Custom H2 Console URL##
spring.h2.console.path=/h2
spring.jpa.show-sql=true

File: src/main/resources/application.properties
Comment (SEM24) [inline src/main/resources/application.properties:1]: For the main props use:
```
spring.profiles.active=prod
server.port=28852
spring.jackson.property-naming-strategy=SNAKE_CASE
spring.datasource.driverClassName=org.h2.Driver
spring.sql.init.data-locations=classpath:data/data.sql
spring.jpa.database-platform=org.hibernate.dialect.H2Dialect

spring.jpa.hibernate.ddl-auto=create-drop
spring.jpa.defer-datasource-initialization=true

**Review by SEM24 — COMMENTED** [review]
If you feel fine with the previous stucture, you can merge it, but in my opinion, this option is looking better. Also you can add extra properties like "application-dev.properties" and add this or **_similar_**:
```
spring.datasource.username=sa
spring.datasource.password=
spring.datasource.url=jdbc:h2:mem:/devDB;DB_CLOSE_DELAY=-1
spring.h2.console.enabled=true
#spring.sql.init.mode=always
#
###Custom H2 Console URL##
spring.h2.console.path=/h2
spring.jpa.show-sql=true

**Pull request merged**